### PR TITLE
Geo region code refactor

### PR DESCRIPTION
**Pull request by kd4001**

### CODE REVIEW

File: src/entities/GeoRegion.re [hunk 1]
```diff
@@ -7,10 +7,21 @@ type t =
...
   | Pune
   | Surat
-  | Jaipur
-  | Kolkata;
+  | Kolkata
+  | Jaipur;
 
-let cities: list(t) = [Mumbai, Bangalore, Delhi, Hyderabad, Chennai, Ahmedabad, Pune, Surat, Kolkata, Jaipur];
+let cities: list(t) = [
+  Mumbai,
```
Comment (Prateek13727) [inline src/entities/GeoRegion.re:14]: arrange the cities in alphabetical order everywhere in this module

File: src/entities/GeoRegion.re [hunk 2]
```diff
@@ -40,9 +51,16 @@ let fromString = (region: string): t =>
...
-  | _ => failwith("No geo-region found for given string")
+  | _ =>
+    ErrorUtils.raiseError(
+      ~path="GeoRegion.re",
+      ~message="No geo-region found for given string",
+      ~value=region,
+    )
   };
 
 let getList = (): list(string) => {
```
Comment (Prateek13727) [inline src/entities/GeoRegion.re:62]: let's have one function that returns a list of cities as list(t). `getAllCities`. The consumer repo can use toString function for string conversion if needed

Comment (Prateek13727) [inline src/entities/GeoRegion.re:62]: replace `getList` with getActiveGeoRegions.  This  function that returns a list all georegions as list(t). The consumer repo can use toString function for string conversion if needed

File: src/entities/GeoRegion.re [hunk 3]
```diff
@@ -7,10 +7,21 @@ type t =
   | Ahmedabad
   | Pune
   | Surat
-  | Jaipur
-  | Kolkata;
+  | Kolkata
+  | Jaipur;
 
-let cities: list(t) = [Mumbai, Bangalore, Delhi, Hyderabad, Chennai, Ahmedabad, Pune, Surat, Kolkata, Jaipur];
+let cities: list(t) = [
```
Comment (Prateek13727) [inline src/entities/GeoRegion.re:13]: not needed

File: src/entities/GeoRegion.re (hunk 1 above)
Comment (kd4001) [reply src/entities/GeoRegion.re:14]: These arranged by release order. So we know what was added last.

**Pull request merged**